client/src/components/pages/ProfileUpdate.tsx

```diff
@@ -228,6 +230,24 @@ const Screen: FC = () => {
     );
   };
 
+  const updateProfileToast = (): void => {
```

Comment:
How about we rename this to `showUpdateProfileToast` for clarity?

Reply:
All right, I'll change the name!